packages/expo-constants/src/ExponentConstants.web.ts

```diff
@@ -65,7 +53,7 @@ export default {
     return false;
   },
   get expoVersion(): string {
-    return getExpoPackage().version;
+    return this.manifest.sdkVersion || '';
```

Comment:
Document that this might be an empty string. Could also make this nullable, since developers should check for that anyway now.